src/upcoming-components/AccordionPanel/AccordionPanel.tsx

```diff
@@ -142,10 +142,7 @@ export const AccordionPanel = ({
         id={ariaControlsVar}
         style={{ height: heightVar }}
       >
-        <div
-          className={styles['accordion__panel-body-inner']}
-          data-accordion-panel
-        >
+        <div className={className} data-accordion-panel>
```

Comment:
This should not be changed

Reply:
Sorry, the `className` prop itself should be removed if no reference in stylesheet